plugins/scaffolder-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-scaffolder-backend",
-  "version": "0.5.1",
+  "version": "0.5.0",
```

Comment:
Nope! 😁

Reply:
I undid the accidental change. I guess I need to remove and redo those commits.  ugh

Reply:
Did what I should have done before. Please forgive my novice mistake.